modules/renter/hostdb/hostweight.go

```diff
@@ -186,7 +186,7 @@ func storageRemainingAdjustments(entry modules.HostDBEntry) float64 {
 // version reported by the host.
 func versionAdjustments(entry modules.HostDBEntry) float64 {
 	base := float64(1)
-	if build.VersionCmp(entry.Version, "1.1.2") < 0 {
+	if build.VersionCmp(entry.Version, "1.1.2") <= 0 {
```

Comment:
I actually pushed this commit into master because it was causing all builds to fail. I'm not sure what the proper fix is. Is the expectation that we'll update this code after every release?